### ISSUE TITLE
Is the new version lacking the back and forward buttons? I can't find them.

### ISSUE THREAD

**Issue by 19007361** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#201]
Thank you for this amazing program, by the way.

**Comment by MarshallOfSound** [comment]
See #198 

Thanks :+1:

**Comment by 19007361** [comment]
Ok, but you haven't given an explanation as to why you removed the buttons.

**Comment by MarshallOfSound** [comment]
Please read that issue I linked you will find a bit where I said this

> Back / Forward missing was a design decision based on a thread somewhere (which I will probably never find again) explaining that if you can't use a website without forward/back navigation buttons the website is doing something wrong. :+1:

 > Bearing in mind that was on a google product forum I tried using GPMDP without the forward / back buttons and I managed suprisingly well. I tried to use them for the first few days and after that not really 

TLDR: I removed them because forward / back buttons are in a browser to navigate between websites, they should not be used to navigate internally inside a website

**Comment by 19007361** [comment]
Thank you, but I really liked them and I see no point in removing them. It was really useful for me.

**Comment by vovokrall** [comment]
I'm missing these buttons too.

**Comment by MarshallOfSound** [comment]
I'm sorry but unless someone can come up with a "very" good reason as to why they should come back.  They aren't coming back

**Comment by 19007361** [comment]
Thanks MarshallOfSound for creating them. I'm just wondering if it's any effort to just _not remove_ the buttons' code that you worked hard on? They were useful for me to jump to another song, add it to my queue, then go back to the previous album I was on. Also, you know, you want to be able to go back to places you just visited, otherwise why are there back and forward buttons on a browser? It was just useful, there was nothing negative about them that I could see.

**Comment by MarshallOfSound** [comment]
Unfortunately it's not as simple as "not removing" the code that did them before.  This app is now based on an entirely new framework and written in a different language.

The use case you describe of going from album to song and then back to album can be achieved through the GPM UI because the album appears as a link next to all songs.

Lack of negatives is not a reason for a feature, it needs to have a positive?  And to me at the moment, the buttons would clutter up the UI and make users navigate through the app in an incorrect way.

**Comment by 19007361** [comment]
OK, I understand. I didn't realize about the new language.
I guess that's fine, it's just that it's been working well for me in Spotify and in your program.
